src/Middleware/UnauthorizedHandler/DefaultRedirectHandler.php

```diff
@@ -108,8 +110,13 @@ protected function createFlashMessage($options): array
     {
         $message = (array)($options['flash'] ?? []);
 
+        $unauthorizedUrl = '';
+        if (Configure::read('Auth.AuthorizationMiddleware.unauthorizedHandler.addUnauthorizedUrlinFlashMessage')){
```

Comment:
You should use local config `$this->getConfig('addUnauthorizedUrlinFlashMessage')`

Reply:
I am suggesting to use debug config instead to avoid adding new configs

Reply:
okay. It make sense.